tests/python/codegen/test_codegen_integration.py

```diff
@@ -117,3 +131,25 @@ def test_gpu_codegen_function_condition(self):
             else:
                 # Not meeting the condition means the agent function will now have executed and the x value should be unchanged
                 assert x == i
+
+    def test_gpu_codegen_line_error(self):
```

Comment:
Test suite additions only cover one of the 3 possible line based error types

- Doesn't check that a `filename.py(LINE)` error is produced
- Dosen't check that a `DynamicPython(LINE)` error is produced